react-events/my-app/src/CustomButton.js

```diff
@@ -0,0 +1,10 @@
+export default function CustomButton({text, color, onCustomClick}) {
+  function handleClick() {
+    onCustomClick(text);
+  }
+  return (
+    <button style={{backgroundColor: color}} onClick={handleClick}>
```

Comment:
Good. You could make it more succinct by inlining the handler:
`onClick={() => onCustomClick(text)}`